src/Core/Flushable.php

```diff
@@ -4,7 +4,7 @@
 
 /**
  * Provides an interface for classes to implement their own flushing functionality
- * whenever flush=1 is requested.
+ * whenever a flush is requested in the kernel.
```

Comment:
Tangentially related change to account for _any_ way we get the kernel to do a flush.